backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientMutationResolver.java

```diff
@@ -35,9 +40,15 @@ public PatientMutationResolver(PersonService ps, UploadService us) {
         _us = us;
     }
 
-    public String uploadPatients(Part part) throws Exception {
-        InputStream people = part.getInputStream();
-        return _us.processPersonCSV(people);
+    public String uploadPatients(Part part) {
+        try (InputStream people = part.getInputStream()) {
+            return _us.processPersonCSV(people);
+        } catch (IllegalGraphqlArgumentException e) {
+            throw e;
+        } catch (Exception e) {
+            LOG.error("Patient CSV upload failed", e);
+            throw new IllegalGraphqlArgumentException("Unable to complete patient CSV upload");
```

Comment:
I like that this conceals the potential leak of an underlying exception 👍 

Did Fortify catch this one with the rest?

Reply:
Yes, sort of. Fortify wanted the checked exception declaration to be tighter than just `throws Exception` and wanted the input stream to always be closed, and if you squint and put those hands together, it means wrapping any IO exceptions in something that would be expected upstream.

`IllegalGraphqlArgumentException` doesn't feel right here, since it would be standing in for an unexpected server error, but we've seem to be using it as the default unchecked exception in this app.

Reply:
We do, but we really shouldn't.

Reply:
Replaced this with a custom `GraphQLError`.